CMSgov/design-system
# [WNMGDS-532] Add CMSDS core link for child design systems

### Added
- Added CMSDS documentation flag to include a link to the core CMSDS 
- Added maturity status badge to all component pages

### Changed
- Updated doc site header to include link to CMSDS for component and pattern pages - this link shows up on the child design system only 

### Screenshot
![Screen Shot 2020-12-04 at 5 02 19 PM](https://user-images.githubusercontent.com/1449852/101219750-7b5f6680-3652-11eb-8e7c-2455bb7bb9b1.png)

### How to test
- setup a child design system in the package.json to this branch with `file:../design-system/package/design-system/`
- run `git clean -fdx` and then `yarn install` Install the local DS branch in the child ds
- Visit component and pattern pages 
- See that pages that are from the core will have the CMSDS link 
- See that pages that are unique to the child design system will not have the CMSDS link
